pages/linux/microcom.md

```diff
@@ -0,0 +1,11 @@
+# microcom
+
+> A minimalistic terminal program.
```

Comment:
This description, while short and to the point, can be a little confusing (I know it's a program that runs in the terminal, but what does it do?). Perhaps it could be expanded to incorporate the fact that it talks to connected devices via a serial connection?

Reply:
I'v expanded the description to make it clear that microcom is used to connect to other devices trough a serial, CAN or telnet connection.